features/support/page_objects/summary_page_error.rb

```diff
@@ -0,0 +1,33 @@
+class SummaryPageError < BasePage
+  set_url '/summary'
+
+  section :content, '#content' do
+    element :step_info, '.govuk-caption-l', text: 'Step 21 of 22'
+    element :header, 'h1', text: 'Check details'
+    element :check_details_hint, 'p', text: 'Please check your details are correct. If you make changes, you may have to answer new questions and confirm information you’ve already entered.'
+    element :probate, '.govuk-summary-list__row', text: 'Probate case No'
+    element :declaration_of_truth, 'h2', text: 'Declaration and statement of truth'
+    sections :summary_row, '.govuk-summary-list__row' do
+      element :action, 'a', text: 'Change'
+    end
+    element :submit_application_button, 'input[value="Submit application and continue"]'
+    element :benefits_change_button, [href: "/questions/benefit"]
+    element :error, ["govuk-error-message", text: "You’ve made changes. Please answer the highlighted questions to complete your application."]
```

Comment:
This and the line above is the only difference to summary_page I can see. And this line is not used anywhere.